packages/babel-plugin-transform-es2015-block-scoping/README.md

```diff
@@ -14,12 +14,24 @@ npm install --save-dev babel-plugin-transform-es2015-block-scoping
 
 **.babelrc**
 
+Without options:
```

Comment:
Could you add a new line before the code block, otherwise it won't be formatted correctly on the website.